ui/theme/partials/navigation.hbs

```diff
@@ -20,7 +20,8 @@
     {{/each}}
 
     <h3 class="nav-title">More</h3>
-    {{#each (pick site.components "community-contracts, upgrades-plugins, contracts-cairo, contracts-stylus, substrate-runtimes, uniswap-hooks") }}
+    {{#each (pick site.components "community-contracts, upgrades-plugins, contracts-cairo, contracts-stylus,
+    substrate-runtimes, stellar-contracts, uniswap-hooks") }}
```

Comment:
It looks like Substrate Runtimes and Stellar Contracts do not appear in the left navigation on the preview: https://deploy-preview-427--openzeppelin-docs.netlify.app/

Reply:
Could this be due to the code being on two lines?  Uniswap Hooks appears though.

Reply:
I tried to run `watch:local` npm command on my local, and was able to identify and fix some of the problems. 

Unfortunately, this left navigation is completely missing when I preview it online, so cannot debug this one

Reply:
weird, now `Stellar Contract` are visible, but `Substrate Runtimes` are still gone 🤷

Reply:
wow yep, it was related to single-line vs double-line. This is also resolved 👍